services/release/release.go

```diff
@@ -165,10 +166,22 @@ func UpdateReleaseOrCreatReleaseFromTag(doer *models.User, gitRepo *git.Reposito
 		return err
 	}
 
-	if err = models.AddReleaseAttachments(rel.ID, attachmentUUIDs); err != nil {
+	if err = models.AddReleaseAttachments(rel.ID, addAttachmentUUIDs); err != nil {
 		log.Error("AddReleaseAttachments: %v", err)
 	}
 
+	if len(delAttachmentUUIDs) > 0 {
+		// Check attachments
+		attachments, err := models.GetAttachmentsByUUIDs(delAttachmentUUIDs)
```

Comment:
this allows me to delete any attachment, without restriction to the current repo!